### DISCUSSION TITLE
JSON: Syntax Error - Malformed JSON

### DISCUSSION THREAD

**Discussion by aavictoria99** [code4recovery/12-step-meeting-list#946] [General]
I get the attached error message when I attempt to add a data source

Another user I know has no problems with same JSON

Help please

![JSON-Error-Message](https://user-images.githubusercontent.com/78445180/194159896-6b71ebd3-a5e8-4c3e-94d2-a9620f30605d.jpg)

**Comment by joshreisner** [comment]
if you put the URL through jsonlint.com or meetingguide.org/validate does it work?

this can happen if the site has sitelock enabled. they have to 'whitelist' your URL to allow it, otherwise it shows a non-JSON error message, triggering the error

**Reply by aavictoria99** [reply]
Thank you Josh.
I have put URL through jsonlint.com and got this response:
![image](https://user-images.githubusercontent.com/78445180/194218555-69186d53-efdf-4ba5-8f21-86e47ec10c2d.png)
I am checking with relevant website IT people
Andrew
